ch02_elf/15_hacking-glibc/use_tls.c

```diff
@@ -0,0 +1,4 @@
+#include <stdio.h>
+#include <threads.h>
+thread_local unsigned long tls_var_with_init = 0xaabbccddaabbccdd;
```

Comment:
```suggestion
_Thread_local unsigned long tls_var_with_init = 0xaabbccddaabbccdd;
```

本文では `_Thread_local` を使っています。

Reply:
ありがとうございます。

Reply:
これ未解決です。

Reply:
う、すみません。Commit suggestion しました。